src/Sylius/Bundle/CoreBundle/DependencyInjection/Configuration.php

```diff
@@ -51,6 +51,12 @@ public function getConfigTreeBuilder(): TreeBuilder
                 ->scalarNode('driver')->defaultValue(SyliusResourceBundle::DRIVER_DOCTRINE_ORM)->end()
                 ->booleanNode('prepend_doctrine_migrations')->defaultTrue()->end()
                 ->booleanNode('shipping_address_based_taxation')->defaultFalse()->end()
+                ->arrayNode('applying_catalog_promotion_on_variants')
```

Comment:
I'm wondering... why not just `catalog_promotions`? 😄 this is obvious we do not apply catalog promotions on anything else than variants and even if one would like to do it - the configuration itself can be used in multiple contexts, not only for variants 💃

Reply:
Additionally - there would be no verb in the configuration, which means better consistency